vnext/Desktop/WebSocketModule.cpp

```diff
@@ -15,16 +15,21 @@ using namespace folly;
 
 using std::string;
 
-namespace Microsoft {
-namespace React {
+namespace {
+
+constexpr char moduleName[] = "WebSocketModule";
```

Comment:
>char moduleName[] [](start = 10, length = 17)

(nit) since we are not dealing with arrays and array sizes, char* type could be a more common option.

Reply:
`constexpr char* moduleName = "WebSocketModule";` threw a compiler error.
Is there a correct way to declare it as `char*` using `constexpr`?

Reply:
just use const char*, it should work.

---
In reply to: [298411580](https://github.com/microsoft/react-native-windows/pull/2662#discussion_r298411580) [](ancestors = 298411580)